rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -220,6 +220,8 @@ namespace rsx
 
 			if (elapsed >= m_update_interval || m_force_update)
 			{
+				m_update_timer.Start();
```

Comment:
Doesn't this need an m_force_update check?

Reply:
Fixed